tests/python/test_atomic.py

```diff
@@ -47,6 +47,33 @@ def test_atomic_add_global_f32():
                                valproc=lambda x: approx(x, rel=1e-5))
 
 
+@ti.test(arch=[ti.cpu, ti.cuda])
+def test_atomic_min_max_uint():
+    x = ti.field(ti.u64, shape=100)
+
+    @ti.kernel
+    def test0():
+        for I in x:
+            x[I] = 0
+        x[1] = ti.cast(1, ti.u64) << 63
+        for I in x:
+            ti.atomic_max(x[0], x[1])
```

Comment:
```suggestion
            ti.atomic_max(x[0], x[I])
```
Is this the original intention?

Reply:
Fixed. I think both are OK.